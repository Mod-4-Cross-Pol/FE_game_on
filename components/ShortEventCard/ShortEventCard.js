import React from 'react';
import { StyleSheet, View, Text} from 'react-native';

export default function ShortEventCard(props) {
  return (
    <View key={Math.random()} style={styles.smallCardContainer}>
      <Text style={styles.titleText}>{props.activity}</Text>
      <Text style={styles.text}>{props.location}</Text>
      <Text style={styles.text}>Time: {props.time}</Text>
    </View>
  );
}

const styles = StyleSheet.create({
  smallCardContainer: {
    height: 100,
    width: 'auto',
    marginBottom: 20,
    backgroundColor: '#28BFBD',
    borderRadius: 50,
    alignItems: 'center',
    borderWidth: 3,
    borderColor: '#3b4aff'
  },
  titleText: {
    fontFamily: 'Farah',
    fontSize: 30,
    color: '#000',
    marginTop: 7
  },
  text: {
    fontSize: 20,
    color: '#000'
  }
});
