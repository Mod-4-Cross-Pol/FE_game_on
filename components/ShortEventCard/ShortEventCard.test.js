import React from 'react';
import { ShortEventCard } from './ShortEventCard';
import renderer from 'react-test-renderer';

describe('ShortEventCard', () => {

  describe('Renders a ShortEventCard component', () => {

    test('renders correctly', () => {
      const wrapper = renderer.create(<ShortEventCard />).toJSON();
      expect(wrapper).toMatchSnapshot();
    });

    test('it works', () => {
      expect(true).toBeTruthy();
    });

  });

});
