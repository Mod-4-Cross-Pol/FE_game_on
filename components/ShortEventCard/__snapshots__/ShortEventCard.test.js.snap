// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShortEventCard Renders a ShortEventCard component renders the component 1`] = `ShallowWrapper {}`;
